ci/properties/webpack.properties.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Webpack",
+    "description": "Build a NodeJS project with npm and webpack.",
+    "iconName": "webpack",
+    "categories": ["JavaScript", "npm", "Webpack"]
```

Comment:
nit: should `npm` be `Npm`?

Reply:
`npm` should be fine, commercially it is known as `npm` only.